pgt/src/PGT/Output/Text.hs

```diff
@@ -55,3 +56,6 @@ parseName = do
           , '_'
           , ' '
           ]
+
+unlines :: (Foldable f, Conversion Text a) => f a -> Text
```

Comment:
__suggestion: (nitpick)__ The `Foldable` type should appear last in the set of types.